src/Controls/Maps/src/Controls.Maps.csproj

```diff
@@ -10,14 +10,14 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <NoWarn>$(NoWarn);CS1591;RS0041;RS0026;RS0027;RS0022</NoWarn>
     <WarningsAsErrors>$(WarningsAsErrors);CS1591</WarningsAsErrors>
+  </PropertyGroup>
 
-    <!-- Add specifics for this NuGet -->
+  <PropertyGroup>
+    <!-- NuGet package information -->
     <IsPackable>True</IsPackable>
-    <RepositoryType>git</RepositoryType>
-    <PackageTags>dotnet-maui;maui;maps;map;ios;android;macos</PackageTags>
-    <Title>.NET MAUI Maps</Title>
-    <Description>.NET MAUI Maps provides a map control for .NET MAUI apps.</Description>
-    <PackageIcon>icon.png</PackageIcon>
+    <PackageId>Microsoft.Maui.Controls.Maps</PackageId>
+    <PackageTags>$(DefaultPackageTags);maps;mapping</PackageTags>
+    <Description>Maps and mapping support for .NET Multi-platform App UI (.NET MAUI) apps.</Description>
```

Comment:
Nit pick: apps again rather than applications. Either are good with me. I'm just pointing out the inconsistency.